From 5e6dc53031853d90b61488df53e089893127c1da Mon Sep 17 00:00:00 2001
From: Chris McGee <newton688@gmail.com>
Date: Tue, 28 Jul 2020 20:16:34 -0400
Subject: [PATCH 1/2] Bump mm-glib dep to 1.10

We'll need to use the new API for fixing the LTE support.
---
 .gitlab-ci.yml  | 10 ----------
 src/meson.build |  2 +-
 2 files changed, 1 insertion(+), 11 deletions(-)

diff --git a/.gitlab-ci.yml b/.gitlab-ci.yml
index 070da72..aa198d2 100644
--- a/.gitlab-ci.yml
+++ b/.gitlab-ci.yml
@@ -8,16 +8,6 @@ before_script:
 
 # Ubuntu 14.04 is not supported, see README for details
 #
-# FIXME: Also disable gtk-doc on Ubuntu 16.04 cause it fails install. Would be nice to get it fixed.
-
-ubuntu-16.04:
-    image: ubuntu:xenial
-    artifacts:
-        when: always
-        name: "xenial-${CI_COMMIT_REF_NAME}"
-        paths:
-            - "${CI_PROJECT_DIR}/build"
-    script: meson -Dgtk-doc=false build && ninja -C build && ninja -C build test && ninja -C build install
 
 ubuntu-18.04:
     image: ubuntu:bionic
diff --git a/src/meson.build b/src/meson.build
index e82245f..7c1b636 100644
--- a/src/meson.build
+++ b/src/meson.build
@@ -32,7 +32,7 @@ sources += [ 'gclue-main.c',
              'gclue-location.h', 'gclue-location.c' ]
 
 if get_option('3g-source') or get_option('cdma-source') or get_option('modem-gps-source')
-    geoclue_deps += [ dependency('mm-glib', version: '>= 1.6') ]
+    geoclue_deps += [ dependency('mm-glib', version: '>= 1.10') ]
     sources += [ 'gclue-modem.c',
 				 'gclue-modem.h',
 				 'gclue-modem-manager.c',
-- 
2.30.2

